### DISCUSSION TITLE
Persist Connector Information even after Kubernetes pods restart

### DISCUSSION THREAD

**Discussion by ArunavNobroker** [strimzi/strimzi-kafka-operator#9343] [Q&A] [answered]
I am currently working on deploying a strimzi kafka connect cluster in GKE. As i was testing it I noticed that everytime the pod restarts the connector in that pod is lost. As i plan to use spot nodes for the connect cluster can someone please help me in how I can configure the cluster such that when a pod is restarted the connectors are restarted as well

**Comment by scholzj** [comment]
What does it mean that the Connector is lost? The connectors are definitely
bormally "persisted" accross rolling updates. But depending on the Strimzi
version, they might not end up rescheduled on the same pod.

So if you somehow loose the connecotr, you should share more info what does
it mean, logs and configs.

**Reply by ArunavNobroker** [reply]
If I delete the pod or the node where the pod is running gets restarted which will happen in spot vm instances, the connector is lost.

**Reply by ArunavNobroker** [reply, marked as answer]
Nevermind when I am creating a connector though kubectl it's being persisted even when pod restarts. Only if I create it though the pod ip it's getting lost
